### PR TITLE
[Dockerfile mode] Highlight strings and ports

### PR DESCRIPTION
**Pull request by ficristo**
- highlight strings
- highlight ports as numbers in expose instruction
- highlight comments as such only when they appear at the start of a line
- consider also ` as a line continuation

### CODE REVIEW

File: mode/dockerfile/dockerfile.js [hunk 1]
```diff
@@ -11,35 +11,64 @@
 })(function(CodeMirror) {
   "use strict";
 
+  var from = "from";
+  var fromRegex = new RegExp("^(\\s*)\\b(" + from + ")\\b", "i");
```
Comment (marijnh) [inline mode/dockerfile/dockerfile.js]: Does changing this from a simple inline regexp to this complicated definition add anything?

Comment (ficristo) [reply mode/dockerfile/dockerfile.js]: It seemed more consitent with the other instructions but I can change it back. Let me know.

Comment (marijnh) [reply mode/dockerfile/dockerfile.js]: It's harmless, I guess, but it seemed a bit of a pointless change.

**Comment by marijnh** [comment]
Thanks! Merged.

**Comment by ficristo** [comment]
Thank you!

**Pull request merged**